js/libs/keycloak-js/src/keycloak.js

```diff
@@ -1009,7 +1009,7 @@ function Keycloak (config) {
         if (token) {
             kc.token = token;
             kc.tokenParsed = jwtDecode(token);
-            kc.sessionId = kc.tokenParsed.session_state;
+            kc.sessionId = kc.tokenParsed.sid;
```

Comment:
Was this `sid` present before? Users might use different versions of Keycloak, so if `sid` was introduced in this PR we should still fall back to `session_state` in case it doesn't exist.

Reply:
`sid` was already present as duplicate of `session_state`, `session_state` is now removed, so if an old adapter is used with a new version of Keycloak (which includes this PR) the `Session State` mapper must be added (documented it in the release changes).

Reply:
Ok, perfect, just wanted to do a sanity check.

Reply:
Thanks!